Source/MediaInfo/Export/Export_Mpeg7.cpp

```diff
@@ -219,6 +219,7 @@ Ztring Mpeg7_ContentCS_Name(int32u termID, MediaInfo_Internal &MI, size_t) //xxy
                         case 1 : return __T("Image");
                         case 2 : return __T("Video");
                         case 3 : return __T("Graphics");
+

```

Comment:
"break" after the "}" and move of the content of default below (default can then be removed) after the switch so any unknown value uses this default.